tools/windows/install-help/cal/customaction.h

```diff
@@ -118,17 +118,17 @@ UINT doUninstallAs(UNINSTALL_TYPE t);
 
 // see https://stackoverflow.com/a/45565001/425565
 // Template ErrType can be HRESULT (long) or DWORD (unsigned long)
-template <class ErrType> std::string GetErrorMessageStr(ErrType errCode)
+template<class ErrType> std::wstring GetErrorMessageStr(ErrType errCode)
 {
     const int buffsize = 4096;
-    char buffer[buffsize];
+    wchar_t buffer[buffsize];
     const DWORD len =
-        FormatMessageA(FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS | FORMAT_MESSAGE_MAX_WIDTH_MASK,
+        FormatMessageW(FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS | FORMAT_MESSAGE_MAX_WIDTH_MASK,
                        nullptr, // (not used with FORMAT_MESSAGE_FROM_SYSTEM)
                        errCode, MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT), &buffer[0], buffsize, nullptr);
     if (len > 0)
     {
-        return std::string(buffer, len);
+        return std::wstring(buffer, len);
     }
-    return "Failed to retrieve error message string.";
+    return L"Failed to retrieve error message string.";
 }
```

Comment:
Would suggest something like 
`L"Failed to retrieve error message %d, original error %d", GetLastError(), errCode"`.  This way, if (for some reason) the format message fails, we don't lose the error number in the logs.